crates/arrow_util/src/pretty.rs

```diff
@@ -548,6 +569,17 @@ impl ColumnValues {
     }
 }
 
+fn fmt_floats(flt: Option<&f64>) -> String {
+    match flt {
+        Some(float) => {
+            //Change here to configure digits after decimalpoint for long floats
+            let str = format!("{:.5}", float);
+            str
```

Comment:
do you need to declare `str` here or can you just return the output of `format`?

Reply:
Same issue with my code here, Will fix it.

Reply:
I saw the sample floats mentioned in issue #1807  to pick 5 digits after decimals, I don't have a solid reason tbh.
I was hoping someone from the team would guide me as to what is required.